index.html

```diff
@@ -7,59 +7,216 @@
+						<div class="frame-2">
+							<span class="client">CANOPY</span>
+							<div class="counter"></div>
+							<span class="role">Back End Dev</span>
+							<div class="counter"></div>
+							<span class="year">2015</span>
+						</div>
+					</div>
+					<p class="works__primary-par">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<div class="works__tags">
+						<span>html</span>
+						<span>css</span>
+						<span>javaScript</span>
+					</div>
+					<a href="#works__card" class="works__action">See Project</a>
+				</section>
+			</article>
+
+			<article class="works__card">
+				<figure class="snapshoot"><img src="./img/Snapshoot-Portfolio3.png" alt=""></figure>
+				<section class="works__left-block">
+					<div class="works__primary-text">
+						<h3 class="works__project-title">Tonic</h3>
+						<div class="frame-2">
+							<span class="client">CANOPY</span>
+							<div class="counter"></div>
+							<span class="role">Back End Dev</span>
+							<div class="counter"></div>
+							<span class="year">2015</span>
+						</div>
+					</div>
+					<p class="works__primary-par">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<div class="works__tags">
+						<span>html</span>
+						<span>css</span>
+						<span>javaScript</span>
+					</div>
+					<a href="#works__card2" class="works__action">See Project</a>
+				</section>
+			</article>
+
+			<article class="works__card">
+				<figure class="snapshoot"><img src="./img/Snapshoot-Portfolio4.png" alt=""></figure>
+				<section class="works__left-block">
+					<div class="works__primary-text">
+						<h3 class="works__project-title">Multi-Post Stories</h3>
+						<div class="frame-2">
+							<span class="client">CANOPY</span>
+							<div class="counter"></div>
+							<span class="role">Back End Dev</span>
+							<div class="counter"></div>
+							<span class="year">2015</span>
+						</div>
+					</div>
+					<p class="works__primary-par">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<div class="works__tags">
+						<span>html</span>
+						<span>css</span>
+						<span>javaScript</span>
+					</div>
+					<a href="#works__card" class="works__action">See Project</a>
+				</section>
+			</article>
+
 		</section>
-		<section class="action">
-			<a href="#" class="down-button">
-				<img src="./img/down_icon.png" alt="down botton"/>
-			</a>
+
+		<section class="about-myself" id="about-myself">
+			<h1 class="about-myself__title">About Myself</h1>
+			<p class="about-myself__primary-par">
+				Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+			<span>LET’S CONNECT</span>
```

Comment:
- [ ] Again well done on the Design of this section 👏👏. The requirements here suggest you should use a heading tag (`hx`)  for the text.
![image](https://user-images.githubusercontent.com/18730106/143712109-a1bd8806-ceee-4394-b4a2-79d4f2ef051d.png)
Please kindly make this minor change Thank you 🙏.
A change like this is all you need thanks.
```suggestion
			<h2>LET’S CONNECT</h2>
```